docs/course_materials/linux_intro.html

```diff
@@ -24,477 +24,635 @@
+            <ul>
+              <li>Creates a directory in the current directory</li>
+            </ul>
+            <pre><code data-trim data-noescape class="hljs lang-python">
+      $ mkdir my_test
+      $
+            </code></pre>
+            <ul>
+              <li>Creates a directory in a specific directory</li>
+            </ul>
+            <pre><code data-trim data-noescape class="hljs lang-python">
+      $ mkdir /home/student/my_test2
+      $
+            </code></pre>
+            <ul>
+              <li>Creates a directory and all parent directories if missing</li>
+            </ul>
+            <pre><code data-trim data-noescape class="hljs lang-python">
+      $ mkdir -p my_test/dir2/dir3/dir4
+      $
+            </code></pre>
+          </section>
+        </section>
+        <section>
+          <section>
+            <h3><b>cd</b></h3>
+            <ul>
+              <li>Change the working directory to another directory existing in the current directory</li>
+            </ul>
+            <pre><code data-trim data-noescape class="hljs lang-python">
+      $ cd my_test
+      $
+              </code></pre>
+          </section>
+
+          <section>
+            <h3><b>/ – The Root Directory </b></h3>
+            <ul>
+              <li>Everything on your Linux system is located under the / directory, known as the root directory.</li>
+            </ul>
+
+            <pre><code data-trim data-noescape class="hljs lang-python">
+      $ cd /
+      $ ls -la
+      bin   dev        etc   lib    lost+found  mnt  proc  run   snap  sys  tmp  var
+      boot  Downloads  home  lib64   media      opt  root  sbin  srv   usr
+
+              </code></pre>
+
+          </section>
+
+          <section>
+            <h3><b> ~ - Home directory </b></h3>
+            <ul>
+              <li>Each Linux user have an home directory, this directory is in /home directory. For example if you have a user called student, then his default home directory is /home/student </li>
+              <li>you can also use ~ to get to your home directory</li>
+
+            </ul>
+
+            <ul>
+              <li>Change Directory to the your user home directory</li>
+            </ul>
+            <pre><code data-trim data-noescape class="hljs lang-python">
+      $ cd
+      $ pwd
+      /home/student
+              </code></pre>
+            <pre><code data-trim data-noescape class="hljs lang-python">
+      $ cd ~
+      $ pwd
+      /home/student
+
+              </code></pre>
+
+          </section>
+
+          <section>
+            <h3><b>Absolute Path vs Relative Path </b></h3>
+            <ul>
+              <li>A path is a unique location to a file or a directory in a file system of an OS.</li>
+              <li>An absolute path is defined as the specifying the location of a file or directory from the root directory(/). In other words we can say absolute path is a complete path from start of actual filesystem from / directory.</li>
```

Comment:
Please rephrase.
(IIRC, I've suggested a phrase on the commit before)

Reply:
I can't find your suggestion, what wrong with this one?

Reply:
Personally, I didn't like 'the specifying the location'.
I think you can make it shorter:
An absolute path starts from the root directory (/) and contains the complete list of directories to locate the specific file or folder.

If you prefer to keep the original:
An absolute path is defined as specifying the location of a file or directory from the root directory(/). In other words, we can say the absolute path is a complete path from the start of the actual filesystem from / directory.

Reply:
I think I prefer the original one.

Reply:
So please follow the minor changes from above